package.json

```diff
@@ -15,14 +15,14 @@
     "astronomy"
   ],
   "dependencies": {
-    "nan": "^2.0.9",
-    "node-gyp": ">=1",
-    "merge": ">=1"
+    "nan": "^2.1.x",
+    "node-gyp": "3.0.x",
+    "merge": "^1.2.x"
   },
   "devDependencies": {},
   "optionalDependencies": {},
   "engines": {
-    "node": ">=0.12"
+    "node": "4.2.x"
```

Comment:
I dont agree that you force the engine to be 4.2, while it is acceptable to use 0.12